CHANGELOG.md

```diff
@@ -1,46 +1,326 @@
-## [Unreleased][unreleased]
+## [0.11.0] - 2017/08/16
+
+The latest and greatest version of Kong features improvements all over the
+board for a better and easier integration with your infrastructure!
+
+The highlights of this release are:
+
+- Support for **regex URIs** in routing, one of the oldest required
+  features from the community.
+- Support for HTTP/2 traffic from your clients.
+- Kong does not depend on Serf anymore, which makes deployment and networking
+  requirements **considerably simpler**.
+- A better integration with orchestration tools thanks to the support for **non
+  FQDNs** in Kong's DNS resolver.
+
+As per usual, our major releases include datastore migrations which are
+considered **breaking changes**. Additionally, this release contains numerous
+breaking changes to the deployment process and proxying behavior that you
+should be familiar with.
+
+We strongly advise that you read this changeset thoroughly, as well as the
+[0.11 Upgrade Path](https://github.com/Mashape/kong/blob/master/UPGRADE.md#upgrade-to-011x)
+if you are planning to upgrade a Kong cluster.
+
+### Breaking changes
+
+##### Configuration
+
+- :warning: Numerous updates were made to the Nginx configuration template.
+  If you are using a custom template, you **must** apply those
+  modifications. See the [0.11 Upgrade
+  Path](https://github.com/Mashape/kong/blob/master/UPGRADE.md#upgrade-to-011x)
+  for a complete list of changes to apply.
+
+##### Migrations & Deployment
+
+- :warning: Migrations are **not** executed automatically by `kong start`
+  anymore. Migrations are now a **manual** process, via the
+  `kong migrations` command. This is to enforce the "single node
+  running migrations" rule.
+  [#2421](https://github.com/Mashape/kong/pull/2421)
```

Comment:
Note to self: This entry doesn't really clarify the reason behind the change and what now needs to be done to run migrations (`kong migrations up`), better update it.